website/pages/blog/judging-open-source-by-github-stars.mdx

```diff
@@ -0,0 +1,254 @@
+---
+title: Judging Open Source by Github Stars
```

Comment:
should we get a more spicy title?
I think about a good title to go on HackerNews...